### PR TITLE
Make it possible to edit the homepage

### PR DESCRIPTION
**Pull request by mattbowen-usds**
- All parts of the homepage are now editable as markdown components. 
- Gatsby's built-in image resizing works on all homepage parts
- We only use one term "link" for URLs used to link from components

### CODE REVIEW

File: tech-far-hub/src/components/initiative.tsx [hunk 1]
```diff
@@ -1,24 +1,25 @@
...
+  const image = getImage(media);
   return (
     <Card layout="flagDefault" headerFirst gridLayout={{ tablet: { col: 12 } }}>
       <CardHeader>
         <h3 className="usa-card__heading">{heading}</h3>
       </CardHeader>
-      <CardMedia exdent>
-        <img src={withPrefix(media)} alt="A stock image" />
-      </CardMedia>
+      <CardMedia exdent>{image && <GatsbyImage image={image} alt="A stock image" />}</CardMedia>
```
Comment (vim-usds) [inline tech-far-hub/src/components/initiative.tsx:22]: This seems cleaner!

File: tech-far-hub/src/pages/index.tsx [hunk 2]
```diff
@@ -1,97 +1,69 @@
...
+          media={node.frontmatter.media_image}
+          media_alt={node.frontmatter.media_alt_text || "An image"}
+        >
+          <div dangerouslySetInnerHTML={{ __html: node.html }} />
         </Initiative>
       );
     }
   });
+  let caseStudyPromo = null;
+  console.log("Case Study", JSON.stringify(data.caseStudy));
```
Comment (vim-usds) [inline tech-far-hub/src/pages/index.tsx:35]: keep console.log()?

File: tech-far-hub/src/pages/index.tsx [hunk 3]
```diff
@@ -1,97 +1,69 @@
...
+  if (
+    data.caseStudy &&
```
Comment (vim-usds) [inline tech-far-hub/src/pages/index.tsx:37]: Not sure if null coalescing or optional chaining might help here?

File: tech-far-hub/src/pages/index.tsx [hunk 4]
```diff
@@ -1,97 +1,69 @@
...
+    data.caseStudy.frontmatter &&
+    data.caseStudy.frontmatter.link &&
+    data.caseStudy.frontmatter.heading &&
+    data.caseStudy.frontmatter.media_alt_text &&
+    data.caseStudy.frontmatter.media_image &&
+    data.caseStudy.html
+  ) {
+    const caseStudyImage = getImage(data.caseStudy.frontmatter.media_image.childImageSharp);
+    console.log("caseStudyImage", caseStudyImage);
```
Comment (vim-usds) [inline tech-far-hub/src/pages/index.tsx:46]: intentional console.log()?

File: tech-far-hub/src/pages/index.tsx [hunk 5]
```diff
@@ -1,97 +1,69 @@
...
-            The VA technology Acquisition Cent, in support of VA OIT/DSVA, utilized FAR 13.1 Simplified Acquisition
-            Procedures, which enabled the team to streamline a multi-step evaluation with onsite demonstrations, and
-            documented the evaluation using a comparative analysis.
-          </p>
-          <StaticImage
-            src="../../static/assets/img/ux-indonesia-8mikJ83LmSQ-unsplash.jpg"
-            width={623}
-            alt="Two people collaborating over graphs"
-          />
+          {caseStudyPromo}
```
Comment (vim-usds) [inline tech-far-hub/src/pages/index.tsx:61]: way to remove code!

**Review by vim-usds — APPROVED** [review]
Curious about why the Gatsby image optimizer was less than ideal?

File: tech-far-hub/src/pages/index.tsx (hunk 2 above)
Comment (mattbowen-usds) [reply tech-far-hub/src/pages/index.tsx]: I need to add a pre-push hook to stop me from leaving them in.

File: tech-far-hub/src/pages/index.tsx (hunk 3 above)
Comment (mattbowen-usds) [reply tech-far-hub/src/pages/index.tsx:36]: It's mostly for the type checker tbh

**Pull request merged**